teachopencadd/talktorials/001_query_chembl/talktorial.ipynb

```diff
@@ -6,19 +6,20 @@
    "source": [
     "# Compound data acquisition (ChEMBL)\n",
     "\n",
-    "Developed in the CADD seminars 2017 and 2018, Volkamer lab, Charité/FU Berlin. \n",
-    "\n",
     "Authors:\n",
-    "- Paula Junge\n",
-    "- Svetlana Leng\n",
-    "- Dominique Sydow, Volkamer lab, Charité"
+    "\n",
+    "- Svetlana Leng, CADD seminars 2017, Volkamer lab, Charité/FU Berlin \n",
```

Comment:
Should it be 'CADD seminar' instead of 'CADD seminars'?

Reply:
Yes, thanks.